src/Quizzer.API.DAL/Entities/Question.cs

```diff
@@ -0,0 +1,19 @@
+﻿using AutoMapper;
+
+namespace Quizzer.API.DAL.Entities {
+    public record Question : EntityBase{
+        enum QuestionType { SingleChoice, MultioChoice, OrderChoice}
```

Comment:
I think enums does not belong to entity classes. But I am not sure, check CB for reference

Reply:
In CB there are some enums in project CookBook.Common. Should i create a new project just because of one enum? There is a possibility to create folder in DAL for enums.

Reply:
I would just for now stick it with just folder in DAL and later on we will see which we need across multiple projects...